src/lightning_app/cli/lightning_cli.py

```diff
@@ -33,10 +34,14 @@
 logger = logging.getLogger(__name__)
 
 
-def get_app_url(runtime_type: RuntimeType, *args) -> str:
+def get_app_url(runtime_type: RuntimeType, *args, **kwargs) -> str:
```

Comment:
This function was messed up from the beginning. The args should be listed explicitly.  
```suggestion
def get_app_url(runtime_type: RuntimeType, *args, need_credits: bool = False) -> str:
```